backend/src/schema/resolvers/embeds.spec.js

```diff
@@ -0,0 +1,216 @@
+import fetch from 'node-fetch'
+import fs from 'fs'
+import path from 'path'
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../../server'
+import { gql } from '../../jest/helpers'
+
+jest.mock('node-fetch')
+const { Response } = jest.requireActual('node-fetch')
+
+afterEach(() => {
+  fetch.mockRestore()
+})
+
+let variables = {}
+
+const HumanConnectionOrg = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/HumanConnectionOrg.html'),
+  'utf8',
+)
+const pr960 = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/pr960.html'),
+  'utf8',
+)
+const babyLovesCat = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/babyLovesCat.html'),
+  'utf8',
+)
+
+const babyLovesCatEmbedResponse = new Response(
+  JSON.stringify({
+    height: 270,
+    provider_name: 'YouTube',
+    title: 'Baby Loves Cat',
+    type: 'video',
+    width: 480,
+    thumbnail_height: 360,
+    provider_url: 'https://www.youtube.com/',
+    thumbnail_width: 480,
+    html:
+      '<iframe width="480" height="270" src="https://www.youtube.com/embed/qkdXAtO40Fo?start=18&feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>',
+    thumbnail_url: 'https://i.ytimg.com/vi/qkdXAtO40Fo/hqdefault.jpg',
+    version: '1.0',
+    author_name: 'Merkley Family',
+    author_url: 'https://www.youtube.com/channel/UC5P8yei950tif7UmdPpkJLQ',
+  }),
+)
+
+describe('Query', () => {
+  describe('embed', () => {
+    let embedAction
+
+    beforeEach(() => {
+      embedAction = async variables => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-22T07:26:41Z" title="Monday, July 22nd 2019, 9:26:41 am +02:00">Jul 22, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,216 @@
+import fetch from 'node-fetch'
+import fs from 'fs'
+import path from 'path'
+import { createTestClient } from 'apollo-server-testing'
+import createServer from '../../server'
+import { gql } from '../../jest/helpers'
+
+jest.mock('node-fetch')
+const { Response } = jest.requireActual('node-fetch')
+
+afterEach(() => {
+  fetch.mockRestore()
+})
+
+let variables = {}
+
+const HumanConnectionOrg = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/HumanConnectionOrg.html'),
+  'utf8',
+)
+const pr960 = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/pr960.html'),
+  'utf8',
+)
+const babyLovesCat = fs.readFileSync(
+  path.join(__dirname, '../../jest/snapshots/embeds/babyLovesCat.html'),
+  'utf8',
+)
+
+const babyLovesCatEmbedResponse = new Response(
+  JSON.stringify({
+    height: 270,
+    provider_name: 'YouTube',
+    title: 'Baby Loves Cat',
+    type: 'video',
+    width: 480,
+    thumbnail_height: 360,
+    provider_url: 'https://www.youtube.com/',
+    thumbnail_width: 480,
+    html:
+      '<iframe width="480" height="270" src="https://www.youtube.com/embed/qkdXAtO40Fo?start=18&feature=oembed" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>',
+    thumbnail_url: 'https://i.ytimg.com/vi/qkdXAtO40Fo/hqdefault.jpg',
+    version: '1.0',
+    author_name: 'Merkley Family',
+    author_url: 'https://www.youtube.com/channel/UC5P8yei950tif7UmdPpkJLQ',
+  }),
+)
+
+describe('Query', () => {
+  describe('embed', () => {
+    let embedAction
+
+    beforeEach(() => {
+      embedAction = async variables => {
```


:heart: the naming convention @roschaefer